easybuild/easyconfigs/r/R/R-3.1.2-foss-2015a.eb

```diff
@@ -0,0 +1,389 @@
+name = 'R'
+version = '3.1.2'
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s']
+
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
+configopts = "--with-lapack --with-blas --with-pic --enable-threads --with-x=no --enable-R-shlib"
+# some recommended packages may fail in a parallel build (e.g. Matrix), and we're installing them anyway below
+configopts += " --with-recommended-packages=no"
+
+dependencies = [
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+    ('libpng', '1.6.16'),  # for plotting in R
+    ('libjpeg-turbo', '1.4.0'),  # for plottting in R
+    ('Java', '1.8.0_25', '', True),  # Java bindings are built if Java is found, might as well provide it
+    ('GMP', '5.1.3'),
```

Comment:
I'd prefer not having this included in here, since it's only needed for a single R package...